bigquery_etl/backfill/utils.py

```diff
@@ -60,7 +61,9 @@ def get_qualified_table_name_to_entries_map_by_project(
     backfills_dict: dict = {}
 
     search_path = Path(sql_dir) / project_id
-    backfill_files = Path(search_path).rglob(BACKFILL_FILE)
+    backfill_files = map(
+        Path, glob(f"{search_path}/**/{BACKFILL_FILE}", recursive=True)
+    )
```

Comment:
`Path.glob()` and `Path.rglob()` will be [getting a new `follow_symlinks` argument in Python 3.13](https://github.com/python/cpython/issues/77609#issuecomment-1567306837), so once we eventually upgrade to that we should be able to revert workarounds like this and specify `follow_symlinks=True` instead.